musicname/ze_ffxii_salikawood_v1_8s.cfg

```
@@ -0,0 +1,21 @@
+"music"
+{
+    "yamamoto kouta - the one.mp3"                                      "The One - Kingdom OST - KOHTA YAMAMOTO"
+    "yamamoto kouta - set off.mp3"                                      "Set Off - Kingdom OST - KOHTA YAMAMOTO"
+    "yamamoto kouta - hung in a bad place.mp3"                          "Hung in a Bad Place - Kingdom OST - KOHTA YAMAMOTO"
+    "studio eim - pvp.mp3"                                              "StudioEIM - Vindictus PVP Theme 3"
+    "studio eim - elchulus the ancient.mp3"                             "StudioEIM - 고대 엘쿨루스 (Elchulus the Ancient)"
+    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega(Epic Dark Orchestral)"
```

Remove extra detail at the end
```suggestion
    "really slow motion & giant apes - vega.mp3"                        "Really Slow Motion & Giant Apes - Vega"
```